website/project/model.py

```diff
@@ -2935,6 +2935,14 @@ def remove_contributors(self, contributors, auth=None, log=True, save=False):
 
         return all(results)
 
+    def move_contributor(self, user, auth, index, save=False):
```

Comment:
We can have an improvement to do a relationship self link for managing just the order of contributors. No reason to enforce that here.

Definitely need a log, though.